install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1602,9 +1602,21 @@ localRedirectPolicy: false
 # To include or exclude matched resources from cilium identity evaluation
 # labels: ""
 
-# logOptions allows you to define logging options. eg:
-# logOptions:
-#   format: json
+# -- (string) Allows you to define logging options.
+# LogOpt sets log driver options for cilium
+# "log-opt", "Log driver options for cilium-health, for example syslog.level=info,syslog.facility=local5,syslog.tag=cilium-agent"
```

Comment:
Is it “for cilium” or “for cilium-health”?

Reply:
it's for cilium, though cilium-health inherits it. a few of these problems are from me copy-pastaing and not cleaning it up. will try to get to it today.